app/livechat/server/api/v1/session.js

```diff
@@ -0,0 +1,131 @@
+import { check, Match } from 'meteor/check';
+
+import { API } from '../../../../api';
+import { Livechat } from '../../lib/Livechat';
+import { LivechatSessions } from '../../../../models';
+
+API.v1.addRoute('livechat/session.incVisitCount/:token', {
+	post() {
+		try {
+			check(this.urlParams, {
+				token: String,
+			});
+
+			const { count } = LivechatSessions.updateSessionCount(this.urlParams.token);
```

Comment:
Let's check if the session exists before updating, if it doesn't, please return `failure` and the reason.
Also, let's use the new method you added  [here](https://github.com/RocketChat/Rocket.Chat/pull/14813/files#diff-60705f83eda16683bd76861a949764d3R33) to find the session.